ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/ClusterPageHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Flex, Skeleton, Title, LabelGroup, Label } from '@patternfly/react-core';
+
+import { gql } from '@apollo/client';
+import { getDateTime } from 'utils/dateUtils';
+
+export const clusterMetadataFragment = gql`
+    fragment ClusterMetadata on Cluster {
+        id
+        name
+        status {
+            orchestratorMetadata {
+                buildDate
+                version
+            }
+        }
+    }
+`;
+
+export type ClusterMetadata = {
+    id: string;
+    name: string;
+    status?: {
+        orchestratorMetadata?: {
+            buildDate?: string; // ISO 8601 date format
+            version: string;
+        };
+    };
+};
+
+export type ClusterPageHeaderProps = {
+    data: ClusterMetadata | undefined;
+};
+
+function ClusterPageHeader({ data }: ClusterPageHeaderProps) {
+    if (!data) {
+        return (
+            <Flex
+                direction={{ default: 'column' }}
+                spaceItems={{ default: 'spaceItemsXs' }}
+                className="pf-u-w-50"
+            >
+                <Skeleton screenreaderText="Loading Cluster name" fontSize="2xl" />
+                <Skeleton screenreaderText="Loading Cluster metadata" height="40px" />
+            </Flex>
+        );
```

Comment:
@sachaudh I'm going to extract this piece to aid in making these header components more reusable like you mentioned in a previous PR. I found that the main header content was too variable to make reusable in a way that made sense, but this piece is a duplicate in each instance.